src/pages/SyntheticsPage/SyntheticsPage.tsx

```diff
@@ -463,7 +489,7 @@ export function SyntheticsPage(p: Props) {
                 [ListSection.Positions]: t`Positions${positionsCount ? ` (${positionsCount})` : ""}`,
                 [ListSection.Orders]: renderOrdersTabTitle(),
                 [ListSection.Trades]: t`Trades`,
-                [ListSection.Claims]: t`Claims`,
+                [ListSection.Claims]: hasClaimables ? t`Claims (1)` : t`Claims`,
```

Comment:
is it okay that `1` is hardcoded?

Reply:
https://app.asana.com/0/home/1205451119040156/1204804284562508
>3. If the user has any Pending Claimable Fees then Claims tab will have an (1) next to it

yes, that's fine